src/main/java/de/tum/in/www1/artemis/repository/FeedbackRepository.java

```diff
@@ -40,6 +42,9 @@
 
     List<Feedback> findByReferenceInAndResult_Submission_Participation_Exercise(List<String> references, Exercise exercise);
 
+    @Query("select feedback from Feedback feedback where feedback.gradingInstruction.id in :gradingInstructionsIds")
+    List<Feedback> findFeedbackCountByStructuredGradingInstructionIds(@Param("gradingInstructionsIds") List<Long> gradingInstructionsIds);
```

Comment:
This name suggests it would return a number ("feedbackCount"), "findFeedbacksBy..." would probably be more intuitive. Or I missunderstood what the method does.

Reply:
thank you for noticing, I was using count before, I just forgot to change method name after returning the Feedback list.